ext/mbstring/mbstring.c

```diff
@@ -861,6 +858,9 @@ static PHP_INI_MH(OnUpdate_mbstring_substitute_character)
 			MBSTRG(filter_illegal_mode) = MBFL_OUTPUTFILTER_ILLEGAL_MODE_CHAR;
 			MBSTRG(current_filter_illegal_mode) = MBFL_OUTPUTFILTER_ILLEGAL_MODE_CHAR;
 			if (ZSTR_LEN(new_value) > 0) {
+				int c;
```

Comment:
If you're moving the declaration anyway, and cleaning it up anyway, might as well merge the declaration and assignment of c?